### PR TITLE
User can scroll to top with a click

### PR DESCRIPTION
**Pull request by kermit-klein**
[PT LINK](https://www.pivotaltracker.com/story/show/173196849)

- Add react-icons package
- Create scrollArrow component
- Scroll button placed on ArticleList and SingleArticle
- Arrow appears after 400 pixel down scroll, on click it scrolls up

Screenshot from 2020-06-06 15-07-28

### CODE REVIEW

**Review by jysmys — APPROVED** [review]

File: cypress/integration/visitorCanClickOnScrollToTopButton.feature.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  it("button is not visible until scrolling down", () => {
+    cy.get(".scrollTop").should('not.be.visible'); 
+  });
+
+  it("button is shown when scrolling down", () => {
+    cy.scrollTo(0, 500);
+    cy.get(".scrollTop").should('be.visible'); 
+  });
+
+  it('succussfully clicks button to scroll up', () => {
```
Comment (erikbjoern) [inline cypress/integration/visitorCanClickOnScrollToTopButton.feature.js:20]: 'succussfully' :'D

File: cypress/integration/visitorCanClickOnScrollToTopButton.feature.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    cy.scrollTo(0, 500);
+    cy.get(".scrollTop").click();
+    cy.wait(3000); 
+    cy.window().then(($window) => {
+      expect($window.scrollY).to.be.closeTo(0, 10);
```
Comment (erikbjoern) [inline cypress/integration/visitorCanClickOnScrollToTopButton.feature.js:25]: Cool.

File: src/css/index.css [hunk 3]
```diff
@@ -40,3 +40,31 @@ a {
...
+}
+
+@keyframes fadeIn {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 0.5;
+  }
+}
```
Comment (erikbjoern) [inline src/css/index.css:74]: Nice.

**Review by erikbjoern — APPROVED** [review]
Good job!

File: cypress/integration/visitorCanClickOnScrollToTopButton.feature.js (hunk 1 above)
Comment (designerofthing) [reply cypress/integration/visitorCanClickOnScrollToTopButton.feature.js:20]: whoops....(native speaker)

**Review by designerofthing — APPROVED** [review]

**Comment by kermit-klein** [comment]
> Good job!

We appreciate it

**Pull request merged**